lib/sles4sap/sap_deployment_automation_framework/deployment.pm

```diff
@@ -55,6 +56,7 @@ our @EXPORT = qw(
   sdaf_deployment_reused
   validate_components
   get_fencing_mechanism
+  sdaf_upload_logs
```

Comment:
Other functions from this file has Unit Testing in `t/20_sdaf_deployment_library.t`

Reply:
Done.